### PR TITLE
Documentation for Cake for Rider release 0.2.0

### PR DESCRIPTION
**Pull request by nils-a**
Documentation and a very small blog post in preparation of the 0.2.0 release of Cake for Rider.

### CODE REVIEW

File: input/docs/integrations/editors/rider/run-configurations.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Order: 30
+Order: 20
```
Comment (pascalberger) [inline input/docs/integrations/editors/rider/run-configurations.md:1]: Why changing order? IntelliSense page has already order 20 (and should be before this page to be consistent with other integration docs)

Comment (nils-a) [reply input/docs/integrations/editors/rider/run-configurations.md:1]: Ah. Because I'm bad at merging things.

Comment (nils-a) [reply input/docs/integrations/editors/rider/run-configurations.md]: @pascalberger I fixed that.

File: input/docs/integrations/editors/rider/run-configurations.md [hunk 2]
```diff
@@ -88,15 +88,29 @@ All settings are project - specific and stored in the `.idea` folder. To share s
...
   This regular expression is used to parse tasks from the Cake files.
   Default: `Task\s*?\(\s*?"(.*?)"\s*?\)`
 * *Verbosity*
-  This is the default verbosity to use, when running a task directly from the tool window or when creating a new run configuration.
+  This is the default verbosity to use, when running a task directly from the Cake tasks window or when creating a new run configuration.
   Default: `normal`
 
+### Search paths
+
+Allows to modify search paths for building the [Cake tasks window](#running-builds-directly).
```
Comment (pascalberger) [inline input/docs/integrations/editors/rider/run-configurations.md:102]: Something seems wrong in this sentence (people are not building the Cake tasks window)

File: input/docs/integrations/editors/rider/run-configurations.md [hunk 3]
```diff
@@ -88,15 +88,29 @@ All settings are project - specific and stored in the `.idea` folder. To share s
 ### Generic settings
 
 * *Cake file extension*
-  This setting is used to find all Cake files and display them in the tool window.
+  This setting is used to find all Cake files and display them in the Cake tasks window.
```
Comment (pascalberger) [inline input/docs/integrations/editors/rider/run-configurations.md:91]:
```suggestion
  This setting is used to find all Cake files and display them in the Cake Tasks window.
```

File: input/docs/integrations/editors/rider/run-configurations.md [hunk 4]
```diff
@@ -88,15 +88,29 @@ All settings are project - specific and stored in the `.idea` folder. To share s
...
   Default: `cake`
 * *Task Regex*
   This regular expression is used to parse tasks from the Cake files.
   Default: `Task\s*?\(\s*?"(.*?)"\s*?\)`
 * *Verbosity*
-  This is the default verbosity to use, when running a task directly from the tool window or when creating a new run configuration.
+  This is the default verbosity to use, when running a task directly from the Cake tasks window or when creating a new run configuration.
```
Comment (pascalberger) [inline input/docs/integrations/editors/rider/run-configurations.md:97]:
```suggestion
This is the default verbosity to use, when running a task directly from the Cake Tasks window or when creating a new run configuration.
```

File: input/docs/integrations/editors/rider/run-configurations.md [hunk 5]
```diff
@@ -109,24 +123,35 @@ Default value is `dotnet-cake` and `dotnet-cake.exe` for Windows.
 ## Creating run configurations
 
 The configurations can either be created from an existing Cake task,
-using the [tool window](#cake-tasks-tool-window) or created manually using the run [configuration editor](#editing-run-configurations).
+using the [cake tasks window](#running-builds-directly) or created manually using the run [configuration editor](#editing-run-configurations).
```
Comment (pascalberger) [inline input/docs/integrations/editors/rider/run-configurations.md:126]:
```suggestion
using the [Cake Tasks window](#running-builds-directly) or created manually using the run [configuration editor](#editing-run-configurations).
```

File: input/docs/integrations/editors/rider/run-configurations.md [hunk 6]
```diff
@@ -109,24 +123,35 @@ Default value is `dotnet-cake` and `dotnet-cake.exe` for Windows.
...
+  Name of the task to run (This is passed to the runner using `--task=<Task>`.)
+* *Verbosity*
+  Verbosity of the Cake output. (This is passed to the runner using `--verbosity=<Verbosity>`.)
+* *Arguments*
+  Arbitrary additional arguments to the runner.
+
+## Running builds directly
 
-The Cake tasks tool window lists all Cake scripts and their tasks:
+The Cake tasks window lists all Cake scripts and their tasks:
```
Comment (pascalberger) [inline input/docs/integrations/editors/rider/run-configurations.md:147]:
```suggestion
The Cake Tasks window lists all Cake scripts and their tasks:
```

File: input/docs/integrations/editors/rider/run-configurations.md [hunk 7]
```diff
@@ -109,24 +123,35 @@ Default value is `dotnet-cake` and `dotnet-cake.exe` for Windows.
...
-![Cake Tasks tool window](/assets/img/cake-rider/docs/toolWindow.png){.img-responsive}
+![Cake Tasks window](/assets/img/cake-rider/docs/toolWindow.png){.img-responsive}
 
 Using a double click on a task will run that task immediately:
 
-![Cake Tasks tool window](/assets/img/cake-rider/docs/cake-run.png){.img-responsive}
+![Cake task run](/assets/img/cake-rider/docs/cake-run.png){.img-responsive}
 
-Alternatively, the buttons at the top of the tool window can be used to either run the task immediately,
+Alternatively, the buttons at the top of the Cake tasks window can be used to either run the task immediately,
```
Comment (pascalberger) [inline input/docs/integrations/editors/rider/run-configurations.md:155]:
```suggestion
Alternatively, the buttons at the top of the Cake Tasks window can be used to either run the task immediately,
or create a new [run configuration](#creating-run-configurations).
```

**Review by pascalberger — CHANGES_REQUESTED** [review]
Added some minor suggestions

**Review by pascalberger — APPROVED** [review]
LGTM

**Comment by pascalberger** [comment]
@nils-a I leave it up to you to merge it once the release is pushed

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by nils-a** [comment]
The release was [approved](https://plugins.jetbrains.com/plugin/15729-cake-for-rider/versions/stable/108104). 
I moved the blog post to "today" and rebased. Will merge as soon as build shows green.

**Pull request merged**